web_widget_image_paste_clipboard/static/src/js/web_widget_image_paste_clipboard.js

```diff
@@ -0,0 +1,35 @@
+odoo.define('web_paste_image.FieldBinaryImage', function(require) {
+    "use strict";
+
+    var core = require('web.core');
+    var FieldBinaryImage = core.form_widget_registry.get('image');
+
+    FieldBinaryImage.include({
+        initialize_content: function() {
+            var self = this;
+            this._super();
+
+            function handlePaste(event) {
```

Comment:
Better define the function as a `FieldBinaryImage` attribute, to allow inheritance.

Reply:
Right now you have:

```js

FieldBinaryImage.include({
  initialize_content: function() {
    function handlePaste(event) {...} // This function is defined under initialize_content, and thus unreachable from the outside world
  }
});
```

I'm asking you to do:

```js
FieldBinaryImage.include({
  init: function () {
    this.events[".o_paste_image paste"] = "handle_paste"; // This is an odoo helper to bind methods to events in widgets
    return this._super.apply(this, arguments);
  },
  handle_paste: function (event) { // Function defined here is extensible and inheritable by other addons
    ...
  }
});
```

Better now? 🍰 😉

Reply:
or.... no need to touch init
```js
 FieldBinaryImage.include({
   events:  _.extends({}, FieldBinaryImage.prototype.events, {
       'paste .o_paste_image': 'handle_paste'
   },
```